docs/mine/spacerace.md

```diff
@@ -187,6 +187,26 @@ lotus-miner sectors status --on-chain-info $SECTOR_NUMBER | grep OnTime
 
 - If the deal is marked as successful on the dashboard but the above commands do not return the expected result, simply repeat steps #1-3 above.
 
+#### How do I force a WaitDeals sector to start sealing?
+
+Run `lotus-miner sectors seal <sector id>`.
+
+#### How is the deal rate of the combined miners being calculated?
+
+The deal rate is calculated based on the sum of all the deals across the miners.
+
+#### If I rent a server in EU from US, can I compete in EU region?
+
+The definition of the `mining operation` can be found [here](https://spacerace.filecoin.io/rules), in short, it is the physical location of the storage and sealing hardware. It might not be cost effective, but you can rent cloud service anywhere in the world. However, it is NOT acceptable to do something like running a proxy or anything of that nature.
```

Comment:
```suggestion
Your region is determined by the physical location of the storage and sealing hardware. It might not be cost effective, but yes, it is permissible to run your mining operation on a rented service anywhere in the world. However, it is NOT acceptable to obscure your miner location via a proxy or similar service."
```

Remove link to website with rules, since this site gets updated more frequently.

Reply:
Can you also move this FAQ to immediately below "How is the "location" of a mining operation determined?"